File: 2/src2/scores0.c
// Generates a bar chart of three scores

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Get scores from user
    int score1 = get_int("Score 1: ");
    int score2 = get_int("Score 2: ");
    int score3 = get_int("Score 3: ");

    // Generate first bar 
    printf("Score 1: ");
    for (int i = 0; i < score1; i++)
    {
        printf("#");
    }
    printf("\n");

    // Generate second bar
    printf("Score 2: ");
    for (int i = 0; i < score2; i++)
    {
        printf("#");
    }
    printf("\n");

    // Generate third bar
    printf("Score 3: ");
    for (int i = 0; i < score3; i++)
    {
        printf("#");
    }
    printf("\n");
}

File: 6/src6/weeks/2/scores0.py
# Generates a bar chart of three scores

from cs50 import get_int

# Get scores from user
score1 = get_int("Score 1: ")
score2 = get_int("Score 2: ")
score3 = get_int("Score 3: ")

# Generate first bar
print("Score 1: ", end="");
for i in range(score1):
    print("#", end="")
print()

# Generate second bar
print("Score 2: ", end="");
for i in range(score2):
    print("#", end="")
print()

# Generate third bar
print("Score 3: ", end="");
for i in range(score3):
    print("#", end="")
print()
